### PR TITLE
Blog post SCM/CI integration roll-out announcement

### PR DESCRIPTION
**Pull request by saraycp**
Can't be merged until the last changes are deployed.

### CODE REVIEW

**Comment by danidoni** [comment]
LGTM

File: _posts/development/2022-09-21-trigger-workflow-rollout.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: SCM/CI Integration in OBS, a Definitive Feature
+category: development
+---
+
+On May 2021 we took our first steps towards the continuous integration between
+OBS and different SCMs. Thanks to the valuable feedback we received from users
+who started as early testers and became heavy users of the feature, we are
+ready to roll it out for every single user in OBS.
```
Comment (hennevogel) [inline _posts/development/2022-09-21-trigger-workflow-rollout.md:10]:
```suggestion
ready today to push it out of the beta program, making it generally available in OBS.
```

File: _posts/development/2022-09-21-trigger-workflow-rollout.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: SCM/CI Integration in OBS, a Definitive Feature
+category: development
+---
+
+On May 2021 we took our first steps towards the continuous integration between
+OBS and different SCMs. Thanks to the valuable feedback we received from users
```
Comment (hennevogel) [inline _posts/development/2022-09-21-trigger-workflow-rollout.md:8]:
```suggestion
In May of 2021 we took our first steps towards putting OBS builds into the continuous integration cycle with the SCM/CI feature. Thanks to the valuable feedback we received from users
```

File: _posts/development/2022-09-21-trigger-workflow-rollout.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: SCM/CI Integration in OBS, a Definitive Feature
```
Comment (hennevogel) [inline _posts/development/2022-09-21-trigger-workflow-rollout.md:3]:
```suggestion
title: General availability of the SCM/CI Integration feature
```

**Review by hennevogel — CHANGES_REQUESTED** [review]
I'm missing a high level description of the feature.

**Comment by saraycp** [comment]
@hennevogel  I introduced your suggestions and added an extra first section describing the feature a bit.

Please have another look.

File: _posts/development/2022-09-21-trigger-workflow-rollout.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+continuous integration cycle with the SCM/CI feature. Thanks to the valuable
+feedback we received from users who started as early testers and became heavy
+users of the feature, we are ready today to push it out of the beta program,
+making it generally available in OBS.
+
+### What Is This About?
+
+With the feature we are pushing out, you can take advantage of source code
+management (SCM) systems like GitHub or GitLab to manage your packages sources
+and then, you can integrate those sources with OBS to run different workflows.
```
Comment (hennevogel) [inline _posts/development/2022-09-21-trigger-workflow-rollout.md:17]:
```suggestion
With the SCM/CI feature you can take advantage of source code management (SCM) systems like GitHub or GitLab to manage your package sources and integrate OBS to run different continuous integration workflows.
```

**Review by hennevogel — CHANGES_REQUESTED** [review]

File: _posts/development/2022-09-21-trigger-workflow-rollout.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+or to create a package in OBS for every software release using Git tags,
+among others.
+
+
+### Haven't You Started With The Integration Yet?
+
+All you need to know is in [our
+documentation](https://openbuildservice.org/help/manuals/obs-user-guide/cha.obs.scm_ci_workflow_integration.html).
+Give it a try!
+
```
Comment (hennevogel) [inline _posts/development/2022-09-21-trigger-workflow-rollout.md:31]: I would add another screenshot of the github side. Like

Screenshot from 2022-09-20 16-23-16

**Review by hennevogel — CHANGES_REQUESTED** [review]

File: _posts/development/2022-09-21-trigger-workflow-rollout.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+<figure>
+  <img src="/images/posts/sprint_126_workflow_runs.png" alt="Workflow runs" />
+  <figcaption>Workflow runs</figcaption>
+</figure>
+
+You can learn more about our journey by reading previous blog posts.
+
+{% include partials/_series-of-posts-about-scm-integration.md %}
```
Comment (hennevogel) [inline _posts/development/2022-09-21-trigger-workflow-rollout.md:39]: I wouldn't add this. It doesn't matter how we got here, it matters that we are here.

**Review by hennevogel — CHANGES_REQUESTED** [review]

File: _posts/development/2022-09-21-trigger-workflow-rollout.md (hunk 5 above)
Comment (saraycp) [reply _posts/development/2022-09-21-trigger-workflow-rollout.md:40]: Done, but I moved the images to _What Is This About?_ section. They make more sense there where we mention workflows and SCM reports.

File: _posts/development/2022-09-21-trigger-workflow-rollout.md (hunk 6 above)
Comment (saraycp) [reply _posts/development/2022-09-21-trigger-workflow-rollout.md]: I don't include the partial on this post then.

File: _posts/development/2022-09-21-trigger-workflow-rollout.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+  <figcaption>Checks results on GitHub</figcaption>
+</figure>
+
+### Haven't You Started With The Integration Yet?
+
+All you need to know is in [our
+documentation](https://openbuildservice.org/help/manuals/obs-user-guide/cha.obs.scm_ci_workflow_integration.html).
+Give it a try!
+
+You can learn more about our journey by reading our previous blog posts about the SCM/CI integration feature.
```
[the remark this answers is not in the document]
Comment (saraycp) [reply _posts/development/2022-09-21-trigger-workflow-rollout.md:41]: No, I decided to leave it there to invite people to read more... But let me remove it and organize the paragraphs differently.

[the remark this answers is not in the document]
Comment (saraycp) [reply _posts/development/2022-09-21-trigger-workflow-rollout.md]: Ok, I removed that sentence and merge the section with the previous one. Let me know what you think, please.

**Review by hennevogel — APPROVED** [review]

**Pull request merged**